modules/lang-painless/src/test/java/org/elasticsearch/painless/action/PainlessExecuteApiTests.java

```diff
@@ -390,11 +390,6 @@ public void testContextWhitelists() throws IOException {
         response = innerShardOperation(request, scriptService, null);
         assertEquals("0beec7b5ea3f0fdbc95d0dd47f3c5bc275da8a33", response.getResult());
 
-        // movfn
-        request = new Request(new Script("MovingFunctions.max(new double[]{1, 3, 2})"), null, null);
-        response = innerShardOperation(request, scriptService, null);
-        assertEquals(3.0, Double.parseDouble((String) response.getResult()), .1);
-
```

Comment:
I moved this to a yaml test in the aggregations module. It only works with that module enabled now.